stack.yaml

```diff
@@ -131,3 +142,11 @@ ghc-options:
   cardano-sl:            -Werror
   cardano-sl-tools:      -Werror
   cardano-sl-lwallet:    -Werror
+  cardano-sl-explorer:   -Werror
+
+# Added for explorer
+flags:
+  cardano-sl:
+    with-explorer: True
```

Comment:
2. No, it's wrong. I even checked it by myself. I cloned your branch, changed default value of `with-explorer` to `False`, used `scripts/build/cardano-sl.sh sl` and it worked fine.

I don't mind if explorer is enabled by default, but I want some changes in build scripts. They are very similar to what I described here: https://serokell.slack.com/archives/G3K2W68AK/p1502200697021298?thread_ts=1502193530.814070&cid=G3K2W68AK
1. You should consider wallet in build script as it's done here: https://github.com/input-output-hk/cardano-sl/pull/1274/files#diff-b4002f72f6a664eaf30722a486b80570 (clean should clean it, it should be possible to build exactly explorer, etc.).
2. `--explorer` should be changed to `--no-explorer` because explorer will be enabled by default. If explorer is built, there should be an error message if `--no-explorer` is passed. But of explorer is not built, `--no-explorer` means that `cardano-sl:-with-explorer` should be used (i. e. `with-explorer` should be disabled).

After that my only concern will be CI which should be discussed with devops I think.